python/rmm/_cuda/gpu.py

```diff
@@ -157,3 +157,22 @@ def deviceGetName(device: int):
     if status != cuda.CUresult.CUDA_SUCCESS:
         raise CUDADriverError(status)
     return device_name.decode()
+
+
+def is_ipc_supported(device: int):
+    """Check if any IPC memory pool handles are supported by the device.
```

Comment:
This isn't scoped to be unique to cudaMallocAsync in any way that I'm missing, is it?

Otherwise I think `is_ipc_supported` is misleading as it sounds like a query if the device supports IPC whatsoever. In reality, it's just querying if you can construct a CUDA memory pool with IPC support.

Reply:
I think we could use the same name/API as in C++: https://github.com/rapidsai/rmm/blob/branch-22.12/include/rmm/detail/dynamic_load_runtime.hpp#L135?

Reply:
Thanks for raising this question -- I struggled with naming here! Do we like `is_async_export_handle_supported`? The difference is that this isn't checking a specific handle type, it's checking if _any_ IPC handle (POSIX=1, Win32=2, Win32Kmt=4) type is supported (otherwise the bitmask returned is 0). Should I refactor everything so that this function takes a handle type and we explicitly check for POSIX handle support in the test suite?

Reply:
@jrhemstad @shwina I went ahead and renamed to `is_async_export_handle_supported`. Alternate suggestions are welcome.